CSL/ClockVisitior.cs

```diff
@@ -0,0 +1,82 @@
+using CSL.Exceptions;
+
+namespace CSL;
+
+using Antlr4.Runtime.Misc;
+using Antlr4.Runtime;
+
+public class ClockVisitor : CSLBaseVisitor<Clock>
+{
+    public override Clock VisitClock(CSLParser.ClockContext context)
+    {
+        try 
+        {
+            if (context == null || context.children == null || context.children.Count != 3)
+            {
+                throw new InvalidLiteralCompilerException("Clock: Invalid syntax, expected format is 'hour:minute'");
```

Comment:
```suggestion
                throw new InvalidLiteralCompilerException($"{nameof(Clock)}: Invalid syntax, expected format is 'hour:minute'");
```

Vi burde bruge "nameof" i sådan nogle situationer her